apps/etterlatte-tidshendelser/src/main/kotlin/no/nav/etterlatte/tidshendelser/omstillingsstoenad/OmstillingsstoenadService.kt

```diff
@@ -27,8 +27,8 @@ class OmstillingsstoenadService(
                 JobbType.OMS_DOED_5AAR -> 60
                 JobbType.OMS_DOED_4MND -> 4
                 JobbType.OMS_DOED_6MND -> 6
-                JobbType.OMS_DOED_10MND -> 9
-                JobbType.OMS_DOED_12MND -> 10
+                JobbType.OMS_DOED_10MND -> 10
```

Comment:
Hva med fristen på oppgavene, blir den riktig også?

Reply:
Ja, har ikke sett noe avvik der for 4MND/6MND-oppgavene